sotodlib/preprocess/processes.py

```diff
@@ -1621,6 +1658,63 @@ def process(self, aman, proc_aman):
         if self.process_cfgs:
             pointing_model.apply_pointing_model(aman)
 
+class BadSubscanFlags(_Preprocess):
+    """Identifies and flags bad subscans.
+
+      Example config block::
+
+        - name : "noisy_subscan_flags"
+            stats_name: tod_stats # optional
+            calc: 
+                nstd_lim: 5.0 
+                merge: False
+            save: True
+            select: True
+    
+    .. autofunction:: sotodlib.tod_ops.flags.get_badsubscan_flags
+    """
+    name = "noisy_subscan_flags"
+
+    def __init__(self, step_cfgs):
+        self.stats_name = step_cfgs.get('stats_name', 'tod_stats')
+        super().__init__(step_cfgs)
+    
+    def calc_and_save(self, aman, proc_aman):
+        if 'flags' not in proc_aman._fields:
+            from sotodlib.core import FlagManager
+            proc_aman.wrap('flags', FlagManager.for_tod(proc_aman))
+        proc_aman.flags.wrap("left_scan", aman.flags.left_scan)
```

Comment:
left_scan and right_scan are already in the proc_aman after FlagTurnarounds, just top-level not under "flags". We probably want to agree on one place for them and just look there